packages/charts/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.base.tsx

```diff
@@ -378,10 +383,7 @@ export class HorizontalBarChartWithAxisBase extends React.Component<
     refArrayIndexNumber: number,
     color: string,
   ): void => {
-    if (
-      this.state.isLegendSelected === false ||
-      (this.state.isLegendSelected && this.state.selectedLegendTitle === point.legend)
-    ) {
+    if (!this._isLegendSelected() || (this._isLegendSelected() && this._isSpecificLegendTitleSelected(point.legend!))) {
```

Comment:
>(!this._isLegendSelected() || (this._isLegendSelected() && this._isSpecificLegendTitleSelected(point.legend!))) { [](http://example.com/codeflow?start=7&length=113)

shouldn't bar hover and bar focus conditions be same?

Reply:
updated